versions/1.0.0.md

```diff
@@ -613,52 +610,39 @@ components:
 }
 ```
 
-#### Reusable Parameter Object
+#### Reusable Object
+
+A simple object to allow referencing of objects contained within the [Components Object](#components-object). It can be used from locations within steps or workflows in the Workflows Description. **Note** - Input Objects MUST use standard JSON Schema referencing via the `$ref` keyword while all non JSON Schema objects use this object and its expression based referencing mechanism.
 
-A simple object to allow reusable parameters from multiple steps or workflows in the Workflows Description.
 
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="reusableParameterName"></a>name | `{expression}` | **REQUIRED**. A [runtime expression](#runtime-expressions) used to reference the desired parameter.
-<a name="reusableParameterValue"></a>value | `string` | A value by default MUST override that of the referenced parameter. 
+<a name="reusableObjectValue"></a>value | `string` | Sets a value of the referenced parameter. This is only applicable for parameter object references.
 
 This object cannot be extended with additional properties and any properties added MUST be ignored.
 
-##### Reusable Parameter Object Example
+##### Reusable Object Example
 
 ```yaml
-  name: $components.parameters.page
-  value: 1
+  reference: $components.successActions.notify
 ```
 
 ```json
   {
-    "name": "{$components.parameters.page}",
-    "value": 1
+    "reference": "{$components.successActions.notify}"
   }
 ```
 
-#### Reusable Object
-
-A simple object to allow referencing of objects contained within the [Components Object](#components-object) from locations within steps or workflows in the Workflows Description. This excludes parameters and inputs defined in the [components/parameters](#components-object). **Note** - parameters should be referenced using the [Reusable Parameter Object](#reusable-parameter-object) and inputs should use standard JSON Schema referencing using the `$ref` keyword.
-
-##### Fixed Fields
-Field Name | Type | Description
----|:---:|---
-<a name="reusableObjectName"></a>name | `{expression}` | **REQUIRED**. A [runtime expression](#runtime-expressions) used to reference the desired object.
-
-This object cannot be extended with additional properties and any properties added MUST be ignored.
-
-##### Reusable Object Example
-
 ```yaml
-  name: $components.successActions.notify
+  reference: $components.parameters.page
+  value: 1
 ```
 
 ```json
   {
-    "name": "{$components.successActions.notify}"
+    "reference": "{$components.parameters.page}",
+    "value": 1

```

Comment:
Suggest we make these consistent - JSON and YAML examples - this implies to me that JSON must have a `{` which is not true. (Same with the example above)